api/server.go

```diff
@@ -73,6 +73,7 @@ type (
 		V2ContractRevisions(id types.FileContractID) (result []explorer.V2FileContract, err error)
 		Search(id string) (explorer.SearchType, error)
 
+		TriggerHostScan(pk types.PublicKey) error
```

Comment:
`nit` triggering (for me) implies a potentially asynchronous action, would rename it to `ScanHost`